### ISSUE TITLE
`testGenerator` in index.js is still empty in 1.2.1

### ISSUE THREAD

**Issue by just4fun** [ember-cli/ember-cli-eslint#52]
I have `"ember-cli-eslint": "1.2.1"` in my `package.json`, but when I installed with `npm install`, the `index.js` in the packages still be the old version.

```javascript
'use strict';
var eslint = require('broccoli-lint-eslint');

module.exports = {
  name: 'ember-cli-eslint',
  included: function (app) {
    this._super.included(app);
    this.jshintrc = app.options.jshintrc;
    this.options = app.options.eslint || {};
  },
  lintTree: function(type, tree) {
    return eslint(tree, {
      testGenerator: this.options.testGenerator || generateEmptyTest
    });
  }
};

// Empty test generator. The reason we do that is that `lintTree`
// will merge the returned tree with the `tests` directory anyway,
// so we minimize the damage by returning empty files instead of
// duplicating app tree.
function generateEmptyTest() {
  return '';
}
```

Any body knows why I always got the old one instead of 87761c7752999d7f85f885e91e859fda061cbde9? Has the new release already been published on npm?

**Comment by morhook** [comment]
I'm having same issue since some weeks now. Looks like for the npm registry has a different commit assigned to 1.2.1 [d7e0c328](https://github.com/jonathanKingston/ember-cli-eslint/commit/d7e0c3281bdd4214d1a072fd8591ec4539861ee1) (I've checked on the downloaded package.json the attribute `gitHead`)

Maybe @BrianSipple can check this one?

**Comment by Turbo87** [comment]
this might be related to #49

**Comment by BrianSipple** [comment]
I commented over in #49 as well, but this is definitely due to the NPM registry using a `gitHead` from a less-than-most-recent commit in `1.2.1`. 

One way to remedy this would be upgrading to `1.3.0`, which just released with some critical updates that -- in my opinion -- make it the first usable version of the project inside of an ember app anyway. 

Regardless, it still makes me a bit sad that this became an issue. If someone has a lead on how to update the `gitHead` that NPM uses for an already-published version, that would be supercool to know going forward.

**Comment by Turbo87** [comment]
@BrianSipple `1.3.0` works great thanks!

About `1.2.1`: Here is how I understand the situation: 

- `1.2.1` was published to NPM 7 months ago, most-likely while at commit https://github.com/jonathanKingston/ember-cli-eslint/commit/d7e0c3281bdd4214d1a072fd8591ec4539861ee1
- As mentioned in https://github.com/jonathanKingston/ember-cli-eslint/issues/49 a few days ago there were no tags for `1.2.1` 
- You tagged https://github.com/jonathanKingston/ember-cli-eslint/commit/4d2781f7b468a95b7545865dd799ff97cb7577b8 as `1.2.1` although that doesn't seem to correspond to the published version on NPM

> update the gitHead that NPM uses for an already-published version

Now I'm confused about your mention of NPM + git. Unless you specifically use it NPM does not care about git tags and isn't downloading from git for the usual version numbers. It just downloads from it's own registry where you need to explicitly publish with `npm publish`, which packs your current folder in a TGZ file and uploads it to their server.

tl;dr: I think the `1.2.1` tag is on the wrong commit and the NPM release should not be touched

**Comment by just4fun** [comment]
@BrianSipple Thank you for 1.3.0.
